ui/apps/platform/src/Containers/Violations/Details/NetworkPolicyModal.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { Button, Flex, Modal, ModalVariant, Title } from '@patternfly/react-core';
+import { CodeEditor, Language } from '@patternfly/react-code-editor';
+
+import CodeEditorDarkModeControl from 'Components/PatternFly/CodeEditorDarkModeControl';
+import { NetworkPolicy } from 'types/networkPolicy.proto';
+import download from 'utils/download';
+
+export type NetworkPolicyModalProps = {
+    networkPolicy: Pick<NetworkPolicy, 'name' | 'yaml'>;
+    isOpen: boolean;
+    onClose: () => void;
+};
+
+function NetworkPolicyModal({ networkPolicy, isOpen, onClose }: NetworkPolicyModalProps) {
+    const [isDarkMode, setIsDarkMode] = useState(false);
+
+    function exportYAMLHandler() {
+        download(`${networkPolicy.name}.yml`, networkPolicy.yaml, 'yml');
+    }
+
+    return (
+        <Modal
+            title="Network policy details"
+            variant={ModalVariant.small}
+            isOpen={isOpen}
+            onClose={onClose}
+            actions={[
+                <Button className="pf-u-display-inline-block" onClick={exportYAMLHandler}>
+                    Export YAML
+                </Button>,
+            ]}
+        >
+            <Flex direction={{ default: 'column' }} spaceItems={{ default: 'spaceItemsSm' }}>
+                <Title headingLevel="h3">{networkPolicy.name}</Title>
```

Comment:
Hmm good catch. I thought that the header for the Modal was a `h2` but it is actually a `h1`.

Better to pass an explicit `header` prop to the `Modal` to force an `h2` header, or change the subheader to a `h2` (and leaving two `h1` elements on the page)? 🤔

Reply:
Off the top of my head, `h2` for the modal seems better for overall page structure.

Let’s discuss as a team to reach consensus and bring all existing up to our standard.

Also worthwhile to ask at Office Hours about accessibility issues and semantic meaning.

Reply:
Here is a different angle on the minor heading independent of levels.

What are pro and con to render instead of heading as form-ish key value pattern at an appropriate size:

**Policy name**: central-db

When I scrolled over the picture again, I noticed central-db is the name of several entities on this page